middleware/near.js

```diff
@@ -44,6 +55,20 @@ async function checkAccountOwnership(ctx, next) {
     return await next();
 }
 
+async function checkAccountDoesNotExist(ctx, next) {
+    const { accountId } = ctx.request.body;
+    let remoteAccount = null;
+    try {
+        remoteAccount = (await ctx.near.account(accountId)).state();
```

Comment:
This should be `await (await ctx.near.account(accountId)).state()`  (as `.state()` returns promise). I guess you just end up catching error from `.account()` call itself. (as it awaits on `.state()` https://github.com/near/near-api-js/blob/40d7f79c19174805e9c7500159f5166819305f5f/src/near.ts#L53).   Then it looks like everything can be reduced to:

```
try {
  await ctx.near.account(accountId);
  ctx.throw(403, 'Account ' + accountId + ' already exists.');
} catch (e) {
  return await next();
}
```